.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
```

Comment:
Why? The original author didn't use semicolons, so why introduce them.

Reply:
If agree if it was inconsistent but it seems that the original author consistently didn't use semicolons. This may just be a default config the pr author just threw in as most people seem to use semicolons.

In this case you really should set this to false as you change far too many lines due to this on top of your feature request which makes it also harder to review.